src/bin/neptune-cli.rs

```diff
@@ -497,6 +496,190 @@ async fn main() -> Result<()> {
         Command::PremineReceivingAddress { network } => {
             return get_nth_receiving_address(*network, args.data_dir.clone(), 0);
         }
+        Command::ShamirCombine { t, network } => {
+            let wallet_dir =
+                DataDirectory::get(args.data_dir.clone(), *network)?.wallet_directory_path();
+            let wallet_file = WalletSecret::wallet_secret_path(&wallet_dir);
+
+            // if the wallet file already exists, bail
+            if wallet_file.exists() {
+                println!(
+                    "Cannot import wallet from Shamir secret shares; wallet file {} already exists. Move it to another location (or remove it) to perform this operation.",
+                    wallet_file.display()
+                );
+                return Ok(());
+            }
+
+            // prompt user for all shares
+            let mut shares = vec![];
+            let capture_integers = Regex::new(r"^(\d+)\/(\d+)$").unwrap();
+            while shares.len() != *t {
+                println!("Enter share index (\"i/n\"): ");
```

Comment:
You probably intended to use `format!` here, to write the values of `i` and `n`, no?